docs/solution-architecture/smartDocumentsIntegration.md

```diff
@@ -45,11 +45,13 @@ Employee->>SmartDocuments: Fill in remaining document data
 Employee->>SmartDocuments: Click "Finish" in the creation wizard
 SmartDocuments->>SmartDocuments: Store document
 SmartDocuments--)+ZAC: Provide document information
-Employee->>SmartDocuments: Close document creation wizard
 
 ZAC->>SmartDocuments: Download document
 ZAC->>+OpenZaak: Link document to case or task
 OpenZaak->>-OpenNotificaties: Send notification that a new document is linked to a case or task
 OpenNotificaties--)ZAC: Received notification "document linked to a case"
 ZAC->>-Behandelaar: Document is zichtbaar in de zaak
+
+ZAC->>Employee: Redirect to result page 
```

Comment:
This is from a different PR/branch which is already merged no? Probably need to rebase. ;-)

Reply:
Nope. It's a new change describing what happens when the file is uploaded:
* 303 to `/static/smart-documents-result.html`
* User has to close the SD tab

Reply:
ah nice!